modules/persistent-storage-fibre-provisioning.adoc

```diff
@@ -32,7 +32,7 @@ spec:
   accessModes:
     - ReadWriteOnce
   fc:
-    wwids: [scsi-3600508b400105e210000900000490000] <1>
+    wwids: [3600508b400105e210000900000490000] <1>
```

Comment:
Totally unrelated to your update, but the monospace items in callout 1 below are in bold for some odd reason. Just in case it's in your area and is something you want to take a look at sometime.